packages/dashboard/webpack.config.js

```diff
@@ -0,0 +1,102 @@
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+const CopyPlugin = require('copy-webpack-plugin');
+
+const mode = process.env.NODE_ENV || 'development';
+
+module.exports = {
+  mode,
+  devtool: mode === 'development' ? 'cheap-module-source-map' : 'source-map',
```

Comment:
Was that intended to add source maps in non-development mode?

Reply:
yes